frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -209,224 +209,210 @@ function roleSelected(evt: any) {
             <div class="row">
                 <form id="grantAccessForm" class="form-container">
                     <div class="row">
-                        <div class="form-group col-md-3 px-0">
-                            <label for="domainInput" class="control-label"
-                                >Select user's domain</label
+                        <label> Select user's domain </label>
```

Comment:
The "for" attributes for the label must be the id of the [input, textarea, button, select, or object elements](https://stackoverflow.com/questions/18945774/validating-html-the-for-attribute-of-the-label-element-must-refer-to-a-form-con). Here we could set the label with "for='idirSelect' or for='becidSelect'", use one of the inputId of he RadioButton component, but I feel it's also okay to just leave it as this. We basically just need a title here, and just use the same label styling.

Reply:
The two radios already have labels associated with the inputs ids previously, so this is like you said a group "title" for the radios. Previously Jota was trying to create a custom class for this special case, but I feel your reasoning on removing "for" attribute is reasonable in this special case.

Reply:
just a question. The No label associated with a form field possible improvements is not a problem?

Reply:
It will show a warning as well, but not an error. We could improve in the future if we have a better idea, but at this moment, I feel there's no need to define a custom class just because of this problem. Thanks!!